.github/workflows/ci.yaml

```
@@ -35,6 +39,23 @@ jobs:
           go-version: 1.24.6
       - name: Build all binaries
         run: make build-all
+  code_coverage:
+    name: Code coverage report
+    needs:
+      - test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v5
+      - name: Post coverage report
+        uses: fgrosse/go-coverage-report@v1.2.0
+        with:
+          coverage-artifact-name: code-coverage
+          coverage-file-name: cover.out
```

Should this not be this?
```suggestion
          coverage-file-name: build/cover.out
```

---

I had this first, too but when we upload the file, the directory gets lost and we only get the downloaded file back.